package-lock.json

```diff
@@ -1,14566 +1,8 @@
 {
   "name": "loast",
   "version": "0.0.0-development",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Should this be lockfileVersion 2 since we're using node 14?

Reply:
no, node 14 still ships with npm v6. We've have to update to 15 or 16 i believe to get npm v6 (which is what produces lockfile version 2)